.eslintrc.yml

```diff
@@ -38,4 +38,37 @@ overrides:
     - doc/**/*.js
     env:
       node: no
-
+  - files:
+    - bin/*
+    - lib/**/*.js
+    rules:
+      no-restricted-globals:
+        - error
+        - name: setTimeout
+          message: &GH-237 See https://github.com/mochajs/mocha/issues/237
```

Comment:
can you explain the `*` and the `&` in the message?

Reply:
`&` defines and names an anchor, and `*` uses it. In this case, you don't have to repeat the same message over and over (DRY).

Reply:
@plroebuck never knew that! awesome